text/setup.py

```diff
@@ -43,6 +43,7 @@
     'autogluon-contrib-nlp==0.0.1b20220208',
     'nlpaug>=1.1.10,<2.0.0',
     'nltk>=3.4.5,<4.0.0'
+    'black~=22.0,>=22.3',
```

Comment:
Let's not add it to text/setup.py. We may add it to Jenkins directly. Users of autogluon.text / AutoMM do not need to install black.

Reply:
Another way around it is add the dependency to text[test] for example. And we can do `pip install text[test]` in Jenkins.